lib/rails_admin_import/formats/json_importer.rb

```diff
@@ -0,0 +1,40 @@
+module RailsAdminImport
+  module Formats
+    class JSONImporter
+      Formats.register(:json, self)
+
+      def initialize(import_model, params)
+        if params.has_key?(:file)
+          @filename = params[:file].tempfile
+        end
+        @import_model = import_model
+      end
+
+      attr_reader :filename, :error
+
+      def valid?
+        if filename.nil?
+          @error = I18n.t('admin.import.missing_file')
+          false
+        else
+          true
+        end
+      end
+
+      # A method that yields a hash of attributes for each record to import
+      def each_record
+        return enum_for(:each_record) unless block_given?
+        
+        File.open(filename) do |file|
+          data = JSON.load(file)
+          if !data.is_a? Array
+            raise ArgumentError, I18n.t('admin.import.data_not_array')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.